frontend/src/components/ScrollToTop/ScrollToTop.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+
+class ScrollToTopOnMount extends Component {
```

Comment:
Does this need to be its own component?

It looks like we could just put the `window.scrollTo` in `Recipes.js`'s `componentDidMount`.

Reply:
This is probably my fault, but :+1: on the change.